.nvmrc

```diff
@@ -1 +1 @@
-18
+22
```

Comment:
Let's bump this. We just develop against the latest version we support to detect regressions early.